Bottom: 05e3b57f8370beac2a47fe65d680f8958dfc07c0
Top:    94fb50a42e9fcae41ff6481b57a6dfd6e8209617
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-13 18:44:27 +0000

Revert "Just make a black background."

This reverts commit cf6c95ade3be149a5a8659bd13a80eedc4092fd2.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index c9255c4..5d2ca7a 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2140,9 +2140,9 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glEnable (GL_STENCIL_TEST);
   glEnable (GL_DEPTH_TEST);
   glDepthFunc (GL_ALWAYS);
-  glClearColor (port->bg_color.red / 65535.,
-                port->bg_color.green / 65535.,
-                port->bg_color.blue / 65535.,
+  glClearColor (port->offlimits_color.red / 65535.,
+                port->offlimits_color.green / 65535.,
+                port->offlimits_color.blue / 65535.,
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
@@ -2255,7 +2255,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* glStencilFunc (GL_GREATER, 1, 1); */           /* Draw only where stencil buffer is 0 */
 
   glDepthMask (GL_FALSE);
-#if 0
   if (global_view_2d) {
     glBegin (GL_QUADS);
     glVertex3i (0,             0,              0);
@@ -2286,7 +2285,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     }
     glEnd ();
   }
-#endif
 
   glDepthMask (GL_TRUE);
